FAIMS/FAIMS3
# Updated hamburger menu

## JIRA Ticket

https://jira.csiro.au/browse/BSS-265
Subtasks : 1 - **https://jira.csiro.au/browse/BSS-295**
                 2 - **https://jira.csiro.au/browse/BSS-296**
                 3 - **https://jira.csiro.au/browse/BSS-297**

## Description

Updated the hamburger menu/side navigation menu to align with our new design specifications. This update presents an opportunity for us to review the menu structure and discuss which options should be included or excluded.


## Proposed Changes

 - New Menu items added to the existing hamburger/side-nav menu
 - Menu items added :
     - Help
     - Switch Org (Have a placeholder based on a check whether you're part of different org)
     - Sign out being directed to where user was directed to. 
- Enhanced styling, added border, shadow to make it look distinct from content page
- Laid out top and bottom menu items as according to Figma design
- Added placeholder routes for switch org and help buttons which can be simply used in future.

## How to Test
Just run the app and you should be able to see the new navigation menu to the left.
If you click on help, switch org, sign-out, you should see a blank meaning the respective components will be connected to them.

## Additional Information

Currently, there is a profile option as per the Figma design. There's some ambiguity regarding what should be included under the profile section so that option hasn't been included in this design.

Notification is not in scope for V1.

For now, I've linked the sign-out option that redirects to the user page where you can sign out which was previously displayed as User.

## Checklist

- [ Y]  I have confirmed all commits have been signed.
- [ Y ] I have added JSDoc style comments to any new functions or classes.
- [ Y ] Relevant documentation such as READMEs, guides, and class comments are updated.
